packages/cbioportal-clinical-timeline/src/Timeline.tsx

```diff
@@ -249,10 +249,12 @@ const Timeline: React.FunctionComponent<ITimelineProps> = observer(function({
     width,
 }: ITimelineProps) {
     const tracks = store.data;
-    const height =
+    const SCROLLBAR_PADDING = 15;
+    let height =
         TICK_AXIS_HEIGHT +
         _.sumBy(tracks, t => t.height) +
-        _.sumBy(customTracks || [], t => t.height(store));
+        _.sumBy(customTracks || [], t => t.height(store)) +
+        SCROLLBAR_PADDING;
```

Comment:
@adamabeshouse this does mean that there will be buffer of white space below timeline when there's no scrollbar.  but that's better than making everything jump when we zoom